doc/Analytics.xml

```diff
@@ -4368,6 +4368,84 @@ xmlns:acme="http://www.acme.com/schema"&gt;
     </tt:ParentTopic>
   </tt:Messages>
 </tt:RuleDescription>
+]]></programlisting>
+    </section>
+    <section>
+      <title>Object Track Aggregation</title>
+      <para>The Object Track Aggregation rule generates an object track aggregation event for below
+        listed scenarios </para>
+      <itemizedlist>
+        <listitem>
+          <para> Optionally, Initial aggregation after an object appears OR an Intermediate aggregation in the field of view 
+            or an Intermediate aggregation while the object is present in the field of view.</para>
+        </listitem>
+        <listitem>
+          <para> Final aggregation when the object is no longer visible in the field of view. </para>
+        </listitem>
+      </itemizedlist>
+      <variablelist role="op">
+        <varlistentry>
+          <term>Parameters</term>
+          <listitem>
+            <para role="param">ClassFilter [tt:StringList]</para>
+            <para role="text">List of classes to be aggregated.</para>
+            <para role="param">ConfidenceLevel - optional [xs:float]</para>
+            <para role="text">Minimum confidence level of object classification for generating
+              aggregation event. </para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Topic</term>
+          <listitem>
+            <para role="param">tns1:RuleEngine/ObjectTrack/Aggregation</para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Source</term>
+          <listitem>
+            <para role="text">See <xref linkend="_msgSource"/></para>
+          </listitem>
+        </varlistentry>
+        <varlistentry>
+          <term>Data</term>
+          <listitem>
+            <para role="param">AggregationStart - mandatory [xs:dateTime] </para>
+            <para role="text">All aggregation events shall set object track start time as the
+              AggregationStart time. </para>
+            <para role="param">AggregationEnd - mandatory [xs:dateTime]</para>
+            <para role="text">AggregationEnd time is usually set to the the last time the object was
```

Comment:
Not sure whether end time is needed at all. 
Isn't end time identical to the event time?

Reply:
We will be preparing to send the event once we collect/prepare the aggregation data and in that sense AggregationEnd time will always be less than or equal to the Event time.
Since its a bit implementation dependent on how close that time difference (Event time - AggregationEnd time) could be, we created this field to make it less ambiguous.

Reply:
Please explain what event time means. From that definition we then can see whether it matches to start, end or something in-between.

Reply:
In my view Event time - the time at which we are sending/posting 'aggregated data' as event.

Reply:
Removed the **AggregationEnd** time from the proposal.